src/cpp/astar.cpp

```diff
@@ -73,6 +96,29 @@ static PyObject *astar(PyObject *self, PyObject *args) {
   nodes_to_visit.push(start_node);
 
   int* nbrs = new int[8];
+  
+  // Get the heuristic method to use
+  float (*heuristic_func)(int, int, int, int, int, int);
+  
+  if (heuristic_override == 1) {
+    heuristic_func = linf_norm;
+  } else if (heuristic_override == 2) {
```

Comment:
Can you make this indentation consistent with the rest of the code please?

Reply:
Sure